java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -382,6 +382,70 @@ public interface ServerManagementClusterCommands {
      */
     CompletableFuture<String> flushall(FlushMode mode, SingleNodeRoute route);
 
+    /**
+     * Deletes all the keys of the currently selected database. This command never fails.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @see <a href="https://valkey.io/commands/flushdb/">valkey.io</a> for details.
+     * @return <code>OK</code>.
+     * @example
+     *     <pre>{@code
+     * String response = client.flushdb().get();
+     * assert response.equals("OK");
+     * }</pre>
+     */
+    CompletableFuture<String> flushdb();
+
+    /**
+     * Deletes all the keys of the currently selected database. This command never fails.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @see <a href="https://valkey.io/commands/flushdb/">valkey.io</a> for details.
+     * @param mode The flushing mode, could be either {@link FlushMode#SYNC} or {@link
+     *     FlushMode#ASYNC}.
+     * @return <code>OK</code>.
+     * @example
+     *     <pre>{@code
+     * String response = client.flushdb(ASYNC).get();
+     * assert response.equals("OK");
+     * }</pre>
+     */
+    CompletableFuture<String> flushdb(FlushMode mode);
+
+    /**
+     * Deletes all the keys of the currently selected database. This command never fails.
+     *
+     * @see <a href="https://valkey.io/commands/flushdb/">valkey.io</a> for details.
+     * @param route Specifies the routing configuration for the command. The client will route the
+     *     command to the nodes defined by <code>route</code>.
+     * @return <code>OK</code>.
+     * @example
+     *     <pre>{@code
+     * Route route = new SlotKeyRoute("key", PRIMARY);
+     * String response = client.flushdb(route).get();
+     * assert response.equals("OK");
+     * }</pre>
+     */
+    CompletableFuture<String> flushdb(SingleNodeRoute route);
```

Comment:
why only single node here? especially since the default is a multinode route

Reply:
- [ ] update to multinode or update comment

Reply:
I copied that from `flushall` - `Route` includes `ALL_NODES` which always fails, because replicas are RO. Even though it may fail now with RANDOM route too.
In a ~dark~ future it is nice to have better route distinuishment by their target - RO and non-RO alongside with single/multi node. In that case we can put here non-RO route to protect a user.

Reply:
What if the users want to send to all primaries? I guess the thinking is they should use the non-route function since it defaults to all primaries?

Reply:
I think yes

Reply:
They have to be smart then.  Thanks for asking Bar - I guess we can either let the user fail (giving them all options) or limit their options but maybe close some edge cases.